src/containers/SubscriptionsPage/reducers.js

```diff
@@ -20,6 +20,6 @@ export const subscriptionsReducer = (state = fromJS({}), action = {}) => {
     case FETCH_SUBSCRIPTIONS_FAILURE:
       return state.merge({ subscriptions: [] });
     default:
-      return state;
+      return state.merge({ subscriptions: [] });
```

Comment:
That makes perfect sense. Also, I'd totally missed the `initialState` declaration above!